apps/sscs/sscs-cor-frontend/demo-image-policy.yaml

```diff
@@ -6,10 +6,10 @@ metadata:
     hmcts.github.com/prod-automated: disabled
 spec:
   filterTags:
-    pattern: '^pr-1009-[a-f0-9]+-(?P<ts>[0-9]+)'
+    pattern: '^pr-1084-[a-f0-9]+-(?P<ts>[0-9]+)'
```

Comment:
Don't think u need this for demo, it can just point to deployed version of demo.